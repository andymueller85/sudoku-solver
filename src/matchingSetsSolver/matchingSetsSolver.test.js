describe('Matching sets solver functions', () => {
  describe('findMatches', () => {
    test.todo('findMatches')
  })

  describe('whittlePossibles', () => {
    test.todo('whittlePossibles')
  })

  describe('processRowMatchingSets', () => {
    test.todo('processRowMatchingSets')
  })

  describe('processColumnMatchingSets', () => {
    test.todo('processColumnMatchingSets')
  })

  describe('processBoxMatchingSets', () => {
    test.todo('processBoxMatchingSets')
  })

  describe('processMatchingSets', () => {
    test.todo('processMatchingSets')
  })
})
