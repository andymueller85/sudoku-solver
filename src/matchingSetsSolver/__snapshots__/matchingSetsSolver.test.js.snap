// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matching sets solver functions findMatches should find all sets of matches for an array of possible values 1`] = `
Array [
  Object {
    "matches": Array [],
    "possibles": Array [
      "9",
    ],
  },
  Object {
    "matches": Array [],
    "possibles": Array [
      "1",
    ],
  },
  Object {
    "matches": Array [],
    "possibles": Array [
      "6",
    ],
  },
  Object {
    "matches": Array [],
    "possibles": Array [
      "4",
    ],
  },
  Object {
    "matches": Array [
      6,
    ],
    "possibles": Array [
      "2",
      "5",
    ],
  },
  Object {
    "matches": Array [],
    "possibles": Array [
      "2",
      "8",
    ],
  },
  Object {
    "matches": Array [
      4,
    ],
    "possibles": Array [
      "2",
      "5",
    ],
  },
  Object {
    "matches": Array [],
    "possibles": Array [
      "3",
    ],
  },
  Object {
    "matches": Array [],
    "possibles": Array [
      "7",
    ],
  },
]
`;
